Tomlet/TomlSerializationMethods.cs

```diff
@@ -187,8 +187,10 @@ internal static Deserialize<object> GetCompositeDeserializer(Type type)
                             var name = field.Name;
                             var m = System.Text.RegularExpressions.Regex.Match(field.Name, "<(.*)>k__BackingField");
                             if (m.Success)
-                                name = m.Groups[1].Value; 
+                                name = m.Groups[1].Value;
 
+                            var name1 = name;
+                            name = type.GetProperties().First(p => p.Name == name1).GetCustomAttribute<TomlPropertyAttribute>()?.GetMappedString() ?? name;
```

Comment:
Please use FirstOrDefault here to not crash if the object being serialized is a field, not a property

Reply:
Changed